mk/help.mk

```diff
@@ -30,6 +30,7 @@ help:
 	@echo "  vrbrain - the VRBrain boards"
 	@echo "  sitl - the SITL Software In The Loop simulation"
 	@echo "  bbbmini - the Beagle Bone Black mini"
+	@echo "  bbblue - the BeagleBone Blue"
```

Comment:
and... Linux boards shouldn't be using make build system as we don't support it anymore.... so you can just drop this patch.